pkg/config/env.go

```diff
@@ -11,18 +11,24 @@ type ConfigMap map[string]interface{}
 type EnvConfig struct {
 	dirs Directories
 
-	AWS       AWSConfig            `toml:"aws"`
-	DockerHub DockerHubConfig      `toml:"dockerhub"`
-	Builders  map[string]ConfigMap `toml:"builders"`
-	Runners   map[string]ConfigMap `toml:"runners"`
-	Daemon    DaemonConfig         `toml:"daemon"`
-	Client    ClientConfig         `toml:"client"`
+	AWS         AWSConfig            `toml:"aws"`
+	DockerHub   DockerHubConfig      `toml:"dockerhub"`
+	Builders    map[string]ConfigMap `toml:"builders"`
+	Runners     map[string]ConfigMap `toml:"runners"`
+	Daemon      DaemonConfig         `toml:"daemon"`
+	Client      ClientConfig         `toml:"client"`
+	TaskStorage TaskStorageConfig    `toml:"taskstorage"`
 }
 
 func (e EnvConfig) Dirs() Directories {
 	return e.dirs
 }
 
+type TaskStorageConfig struct {
+	Path string `toml:"path"`
```

Comment:
I don't think we need that - we can just hardcode under the work directory for now, no?

Reply:
True. If we want to change the storage path (i.e. to a persistent volume), we can just  move $TESTGROUND_HOME.